app/controllers/application_controller.rb

```diff
@@ -1,5 +1,10 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
+
+  def after_sign_in_path_for(resource)
```

Comment:
I added this because devise automatically directs to root after sign-in. just changed it so it goes to socks index

Reply:
ok yeah cool